diff --git a/node_modules/sucrase/dist/transformers/RootTransformer.js b/node_modules/sucrase/dist/transformers/RootTransformer.js
index ced0a17..1c2b359 100644
--- a/node_modules/sucrase/dist/transformers/RootTransformer.js
+++ b/node_modules/sucrase/dist/transformers/RootTransformer.js
@@ -195,7 +195,8 @@ var _TypeScriptTransformer = require('./TypeScriptTransformer'); var _TypeScript
   }
 
   processClass() {
-    const classInfo = _getClassInfo2.default.call(void 0, this, this.tokens, this.nameManager, this.disableESTransforms);
+    // Needed until addons-linter supports class fields.
+    const classInfo = _getClassInfo2.default.call(void 0, this, this.tokens, this.nameManager, false);
 
     // Both static and instance initializers need a class name to use to invoke the initializer, so
     // assign to one if necessary.
